morph/react-native/block-wrap.js

```diff
@@ -12,6 +12,7 @@ export let enter = (node, parent, state) => {
     node.ensureBackgroundColor = true
   }
 
+  let isKeyboardAvoidingView = getProp(node, 'isKeyboardAvoidingView')
```

Comment:
should we add the `isKeyboardAvoidingView` prop to the ignored props list in web?